dashboard/src/shared/url.ts

```diff
@@ -80,8 +80,10 @@ export const backend = {
       backend.apprepositories.base(cluster, namespace),
     list: (cluster: string, namespace: string) => backend.apprepositories.base(cluster, namespace),
     validate: (cluster: string) => `${backend.apprepositories.base(cluster, "kubeapps")}/validate`,
-    delete: (cluster: string, name: string, namespace: string) =>
+    delete: (cluster: string, namespace: string, name: string) =>
```

Comment:
you have changed the order of the parameters here in this `delete` but not in the places it's used, maybe you should leave this change out of this PR.

Reply:
Totally agree, it was wrongly staged and committed while working on #2147. My fault!